junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ResourceLock.java

```diff
@@ -43,12 +46,47 @@ default void close() {
 		release();
 	}
 
+	/**
+	 * {@return the exclusive resources this lock represents}
+	 */
+	List<ExclusiveResource> getResources();
+
+	/**
+	 * {@return whether this lock requires exclusiveness}
```

Comment:
😁 I see you've readily adapted the new Javadoc tag.

Reply:
Indeed! :slightly_smiling_face: